modules/editor/inserter/index.js

```diff
@@ -0,0 +1,22 @@
+const h = wp.element.createElement;
```

Comment:
h because this syntax is inspired by 'hyperscript' https://github.com/hyperhype/hyperscript